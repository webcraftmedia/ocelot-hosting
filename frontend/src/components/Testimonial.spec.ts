
import { renderSuspended } from '@nuxt/test-utils/runtime'
import Component from './Testimonial.vue'
import { describe, expect, it } from 'vitest'

describe('Testimonial', () => {
  it('renders', async () => {
    const html = await (await renderSuspended(Component)).html()
    expect(html).toMatchSnapshot()
  })
})