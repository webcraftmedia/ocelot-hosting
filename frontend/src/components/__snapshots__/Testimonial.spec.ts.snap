// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Testimonial > renders 1`] = `
"<div id="test-wrapper">
  <figure data-v-63dfa6cb="">
    <blockquote data-v-63dfa6cb="">
      <p data-v-63dfa6cb=""></p>
    </blockquote>
    <figcaption data-v-63dfa6cb=""><img data-v-63dfa6cb="" alt="profile picture">
      <div data-v-63dfa6cb="" class="person">
        <div data-v-63dfa6cb="" class="name"></div>
        <div data-v-63dfa6cb="" class="position"></div>
      </div>
    </figcaption>
  </figure>
</div>"
`;
